test/test_classifiers.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import numpy as np
+
+import fake_news.base as base
+from fake_news.classifiers import (
+    NaiveBayes,
+    CNN,
+    logisticRegression,
+    LSTM,
+    RandomForest,
+    RNN,
+    SVM,
+)
+
+
+class TestClassifier(unittest.TestCase):
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.mockInputMl = np.zeros((1, 10000))
+        self.mockInputDl = np.zeros((1, 500))
+
+    def _singleClassifierTest(
+        self, classifier: base.AbstractNewsClassifier, input: np.ndarray
+    ) -> None:
+        pred = classifier.predict(input)
+        self.assertEqual(pred.shape, (1,))
+
+    def testNaiveBayesOutputShape(self):
+        model = NaiveBayes.MultinomialNaiveBayesClassifier(metrics=[])
+        model.load_model("fake_news/classifiers/naivebayes.pkl")
```

Comment:
Is `load_model` needed here - since it isn't supposed to change the input/output sizes?
Either way, its probably a bad idea to rely on persisted data in unit tests 🤷 

Maybe for this project, it doesn't matter that much

Reply:
This project probably doesn't even need unit tests, so I'm only adding them to get more points. I really have no idea what else could be added to the tests